lib/rex/stopwatch.rb

```diff
@@ -7,7 +7,7 @@ module Rex
   # @yield [] The block whose operation should be timed.
   #
   # @return Returns the result of the block and the elapsed time in seconds.
-  def self.stopwatch
+  def self.elapsed_seconds
```

Comment:
Potential race condition, just wanted to confirm the suggestion of using `elapsed_time` here had been seen - https://github.com/rapid7/rex-core/pull/17#discussion_r749763550

```
  def self.elapsed_time(unit: :float_second)
    start = Process.clock_gettime(Process::CLOCK_MONOTONIC, unit)
    ret = yield
    elapsed = Process.clock_gettime(Process::CLOCK_MONOTONIC, unit) - start

    [ret, elapsed]
  end
````

There's a link to the comment above that suggestion in the PR description, but wasn't sure if the latest one had been seen 😄

Reply:
Implemented now, thanks! :+1: